EM v1.0 - resources (#21821)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/accesspackage-delete-resourcerolescopes.md

---

> title: "Remove resourceRoleScope from resourceRoleScopes"
> description: "Remove a resourceRoleScope from the resourceRoleScopes of an access package."
> ms.localizationpriority: medium
> author: "markwahl-msft"
> ms.prod: "governance"
> doc_type: "apiPageType"
> ---
> 
> # Remove resourceRoleScope from resourceRoleScopes

Reviewer: ```suggestion
title: "Remove resourceRoleScope from an accessPackage"
description: "Remove a resourceRoleScope from an access package."
ms.localizationpriority: medium
author: "markwahl-msft"
ms.prod: "governance"
doc_type: "apiPageType"
---

# Remove resourceRoleScope from an accessPackage
```

Author: updated
